openpathsampling/features/numpydocparse.py

```diff
@@ -0,0 +1,145 @@
+class NumpyDocParser(object):
```

Comment:
Docstring, with explanation, please (oh, the irony!). Do I understand correctly that this generates a docstring for snapshot classes? I see how that can be useful from the standpoint of interactive users, but does that mean anything when sphinx generates docs for web/print?

Also, the word "parser" usually implies that it *reads* that thing -- but this looks to me like it is mainly intended for writing (or maybe joining) docstrings. Not sure that "parser" is the right term for this.